include/oneapi/tbb/blocked_rangeNd.h

```diff
@@ -63,20 +60,35 @@ class blocked_rangeNd_impl<Value, N, detail::index_sequence<Is...>> {
 private:
     //! Helper type to construct range with N tbb::blocked_range<value_type> objects.
     template<std::size_t>
-    using dim_type_helper = tbb::blocked_range<value_type>;
+    using dim_range_type = tbb::blocked_range<value_type>;
```

Comment:
Is public in the specification proposal.

Reply:
Indeed needs to be public, as it is used in the constructor.